vital/types/protobuf/activity_type.proto

```diff
@@ -0,0 +1,37 @@
+/*ckwg +29
+ * Copyright 2018-2020 by Kitware, Inc.
```

Comment:
Shouldn't this just be 2020?

Reply:
If it is materially copied from something else (probably how this happened in the first place), not necessarily.

That said, I've asked before (and never gotten any response) whether we should even be bothering or if we should switch to a much shorter statement to the effect of "read the license" and stop trying to maintain copyright years in files altogether.

Reply:
> If it is materially copied from something else (probably how this happened in the first place), not necessarily.
As in, copied from the detected_object_type protobuf? That would make sense.


> That said, I've asked before (and never gotten any response) whether we should even be bothering or if we should switch to a much shorter statement to the effect of "read the license" and stop trying to maintain copyright years in files altogether.

I would certainly be willing to help with this change

Reply:
> As in, copied from the detected_object_type protobuf?

Indeed; according to git, the provenance is `detected_object_type.proto` → `class_map.proto` → {`detected_object_type.proto`, `activity_type.proto`}... which seems fairly likely.

> I would certainly be willing to help with this change

In theory, the presence of `ckwg` tags should make it possible to almost fully automate such a change. The real issue is getting folks to care enough to decide on a direction.

Reply:
Right. Might be worth starting an email thread. I'm going to mark this as resolved for now, but we should continue with this.